app/editor/src/features/content/form/ContentForm.tsx

```diff
@@ -678,6 +678,12 @@ const ContentForm: React.FC<IContentFormProps> = ({
                                       }
                                     />
                                   </div>
+                                  <FormikCheckbox
+                                    label="Is CBRA Unqualified"
+                                    name="isCBRAUnqualified"
+                                    checked={props.values.isCBRAUnqualified}
```

Comment:
You shouldn't need this `checked` property since you're just using the basic checkbox functionality.

Reply:
after removed it, checkbox doesn't work for some reason.